sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSampleDao.java

```diff
@@ -358,25 +358,23 @@ public boolean aliasExists(String alias) throws IOException {
 
   /**
    * Lazy-gets samples associated with a given Project
-   * 
+   *
    * @param Long
    *          projectId
    * @return Collection<Sample> samples
    */
   @Override
   public Collection<Sample> listByProjectId(long projectId) throws IOException {
-    Query query = currentSession().createQuery("from SampleImpl where project.id like :id");
-    query.setLong("id", projectId);
     @SuppressWarnings("unchecked")
-    List<Sample> records = query.list();
+    List<Sample> records = currentSession().createCriteria(SampleImpl.class).add(Restrictions.like("project.id", projectId)).list();
```

Comment:
now you mention it, using like is unnecessary - as evidenced by the lack of %'s.  I shall remove.